File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/search/SearchMvpView.java
package com.mifos.mifosxdroid.online.search;

import com.mifos.mifosxdroid.base.MvpView;
import com.mifos.objects.SearchedEntity;

import java.util.List;

/**
 * Created by Rajan Maurya on 06/06/16.
 */
public interface SearchMvpView extends MvpView {

    void showUserInterface();

    void showSearchedResources(List<SearchedEntity> searchedEntities);

    void showNoResultFound();

    void showMessage(int message);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/search/SearchMvpView.kt
package com.mifos.mifosxdroid.online.search

import com.mifos.mifosxdroid.base.MvpView
import com.mifos.objects.SearchedEntity

/**
 * Created by Rajan Maurya on 06/06/16.
 */
interface SearchMvpView : MvpView {
    fun showUserInterface()
    fun showSearchedResources(searchedEntities: MutableList<SearchedEntity>)
    fun showNoResultFound()
    fun showMessage(message: Int)
}